docs/work/content/04_content/einleitung.tex

```diff
@@ -17,8 +17,14 @@ \chapter{Einleitung}
 	Insgesamt soll ein digitaler Reiseführer entstehen, der das Reiseerlebnis auf eine bessere, digitalere und einfachere Ebene hebt und noch mehr Spaß am Reisen erzeugen kann.
 
 	\section{Motivation}
-	(Joshua)
+
 	Wir möchten mit dieser Arbeit einen Beitrag zu einer digitalisierten und technisch geprägten Gesellschaft leisten, indem wir eine Anwendung schaffen, die mit den Nachteilen von gedruckten Reiseführern aufräumt und eine neue innovative Art des Reisen schafft. Damit soll das Reiseerlebnis der Menschen verbessert werden und ihnen die Möglichkeit geben sich noch mehr auf das Erlebte zu konzentrieren, ohne Gedanken an eine komplizierte Planung, bei der viele Medien parallel genutzt werden, zu verschwenden.
 	Außerdem ist diese Arbeit Teil unserer Prüfung zum Bachelor of Science und dient zur praktischen Anwendung der bisher im Studium erlernten Fähigkeiten und zum Ausprobieren neuer innovativer Techniken um unser Wissen zu erweitern.   
 
-	\section{Ausblick auf die Arbeit}
\ No newline at end of file
+	\section{Ausblick auf die Arbeit}
+	
+	In dieser Arbeit soll die durchgeführte Entwicklung detailliert beschrieben und festgehalten werden. Dazu werden alle wichtigen Arbeitsschritte aufgeschlüsselt und die Ergebnisse entsprechend dargestellt. Die entwickelte App trägt den Namen \textit{Travlyn} und wird im Verlauf dieser Arbeit nur noch so bezeichnet. Dieser Name entstand bei Diskussionen der Teammitglieder und wurde für markenfähig befunden. 
+	
+	\vspace{0.25cm}
+	
+	An erster Stelle steht die Beschreibung der verwendeten Techniken, Prozessen und Begriffen. An dieser Stelle werden alle, für diese Arbeit, relevanten Eigenschaften genannt und Verweise auf weiterführende Literatur geliefert um ein gemeinsame Ausgangslage zu schaffen. Anschließend werden die Anforderungen an die zu entwickelnde App geschildert und Ziele gesetzt, die erreicht werden sollen. Danach wird auf ein sehr zentrales Thema dieser Entwicklung eingegangen: Die Datenquellen. Ein Reiseführer lebt von den präsentierten Daten und somit ist die Auswahl der Datenquellen anhand der Anforderungen von großer Bedeutung. Ein ähnliches Gewicht hat das erstellen eines Übergeordneten Konzepts für die Anwendung und dessen Spezialisierung auf kleinere Komponenten der Software. Das Ergebnis dieses Prozesses wird im Anschluss an die Datenquellen ausführlich dargestellt. Nach der Konzeption folgt die Durchführung und Implementation, von der nur einige zentrale und/oder kritische Punkte dargestellt werden. Abschließend soll das entstandene Ergebnis anhand bestimmter Verfahren evaluiert werden und gegen die zu Beginn definierten Ziele validiert werden. Nach dieser Evaluation mündet diese Arbeit in ein Fazit, welches einen Ausblick auf zukünftige Erweiterung und Verbesserungen gibt und zentrale Punkte, die das Team aus diesem Projekt mitgenommen hat, darstellt.  
```

Comment:
2x Stelle hintereinander
Eigenschaften passt nicht so richtig, vielleicht besser _Technologien_ oder _technischen Grundlagen_
[...] geliefert**,** um **eine** gemeinsame Ausgangslage [...] 
[...] hat das **E**rstellen eines [...] 
[...] Verfahren evaluiert und gegen die zu Beginn definierten Ziele validiert werden. [...] 
[...] Evaluation mündet diese Arbeit in **einem** Fazit [...]